### ISSUE TITLE
More Options?

### ISSUE THREAD

**Issue by runeimp** [oz/tz#17]
Since `-q` is _coming to a future release near you_ can we also add `-v` and/or `-version/--version` so we can check what version is installed without having to dig into the package manager, please? I'd ask for `-h` and `-help/--help` but you get that for free with the `flag` package.  👼

**Comment by mitchblank** [comment]
While we're spitballing, I think that it would also be a better-behaved CLI tool if there was a way to specify the timezones to print on the command line.

It's great that you can specify a default set via `$TZ_LIST` but it would be nice if you could override on the command line.

**Comment by oz** [comment]
Yes, I think we could have the program check its arguments, and use that as a list of time zones, e.g.:

```
tz America/Denver Africa/Bamako
```

While keeping the possibility of aliases:

```
tz America/Denver,foo Africa/Bamako,bar
```

When these are set, the `TZ_LIST` variable is ignored. It doesn't solve the discoverability issue of tzdb zone names, but it takes less keystrokes to change/test things.

**Comment by djmattyg007** [comment]
If nothing else, the `TZ_LIST` environment variable should be mentioned in the help text.

**Comment by oz** [comment]
Fixed in [0.6](https://github.com/oz/tz/releases/tag/v0.6).
